### DISCUSSION TITLE
groups.io user group?

### DISCUSSION THREAD

**Discussion by gm8arv** [AlexandreRouma/SDRPlusPlus#95] [Ideas]
Is there a groups.io group for SDR++ please?  I think it would be a useful addition.  I couldn't find either SDRplusplus or SDR++ in the group search.

Thanks,
David GM8ARV

**Comment by AlexandreRouma** [comment]
Hello,
There currently isn't a group.io, hadn't thought of it.
We currently have a [discord server](https://discord.gg/aFgWjyD) for support and development.

If there is enough interest I'll create a groups.io group for it :)

Regards,
Alex

**Reply by lawendel** [reply]
HI, I'm having issues with my newly created Discord account and would love the simplicity of a classic groups.io discussion environment! I wonder if detailed building instructions for MacOS (x86) will be provided. Also, do you plan to support building via Mac Ports? Thank you! - Andy

**Reply by AlexandreRouma** [reply]
First of all sorry for the late replied.

I looked more into groups.io and sadly they require a fee for groups with over 100 members.
I think an mailing list or IRC (freenode #sdrpp) would be more appropriate.

The macOS build instructions aren't yet available, I'm still dealing with a lot of issues on OSX atm.

Best regards,
Alex

**Reply by gm8arv** [reply]
A mailing list would be fine with me.
David
